opm/output/eclipse/EclipseIO.cpp

```diff
@@ -221,7 +221,7 @@ EclipseIO::Impl::Impl( const EclipseState& eclipseState,
 
 
 
-void EclipseIO::Impl::writeINITFile( const data::Solution& simProps, const NNC& nnc) const {
+void EclipseIO::Impl::writeINITFile( const data::Solution& simProps, std::map<std::string, std::vector<int> > map, const NNC& nnc) const {
```

Comment:
The `Impl`function can just take a `const std::map<std::string, std::vector<int>>&` argument; observe that the main external function should be called with value semantics (as it is currently) - to allow calling scopy to dispatch this in a separate execution context.